### DISCUSSION TITLE
Support rebase by comments @renovate

### DISCUSSION THREAD

**Discussion by Goooler** [renovatebot/renovate#20166] [Suggest an Idea]
### What would you like Renovate to be able to do?

If we can trigger a rebase by commenting something like `@renovate rebase` or `@renovate-bot rebase` like what @dependabot did, that would be much useful.

### If you have any ideas on how this should be implemented, please tell us here.

Example: https://github.com/LawnchairLauncher/lawnchair/pull/3294#issuecomment-1413031698.

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
In future, please start with an ideas discussion first. I have converted this one.

The same PR you link to has this capability instead:

<img width="425" alt="image" src="https://user-images.githubusercontent.com/6311784/216243170-dbcc8de0-6a77-4a35-81f1-9f9e4501e049.png">

**Reply by Goooler** [reply]
Yep, I see it, but it might be better if we can also trigger rebase by commenting.

**Reply by rarkins** [reply]
There are multiple reasons why a checkbox is more efficient for Renovate, so this feature is not planned

**Reply by HonkingGoose** [reply]
> There are multiple reasons why a checkbox is more efficient for Renovate, so this feature is not planned

@rarkins If checkboxes are better for Renovate, do you still want to do this issue?

- https://github.com/renovatebot/renovate/issues/1355

**Reply by rarkins** [reply]
I would prefer a checkbox for merging first. Maybe like we discussed there, we could make comments opt in and with disclaimers

**Reply by HonkingGoose** [reply]
> I would prefer a checkbox for merging first. 

@rarkins I found this old issue about checkboxes for merging:

- https://github.com/renovatebot/renovate/issues/3172

We closed the issue but _didn't_ add the feature. Do you want to re-open that issue (again), or do you want me to create a new issue?

**Reply by rarkins** [reply]
New issue. The last I recall discussing it was that the functionality should be opted into by config, because then users without commit access to the base branch could essentially "tell" Renovate to merge something to that same branch

**Reply by HonkingGoose** [reply]
Here's your new issue:

- #20376
